### PR TITLE
fix: add and remove users in a team

### PR DESCRIPTION
**Pull request by pranita09**
### Describe your changes:

Fixed the Add and Remove user APIs. 
Also added e2e test for Remove user from the table.

https://github.com/user-attachments/assets/f9fe90c6-3051-4cb9-917c-846e0021faa0

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/pages/TeamsPage/TeamsPage.tsx [hunk 1]
```diff
@@ -217,7 +219,6 @@ const TeamsPage = () => {
     try {
       const data = await getTeamByName(name, {
         fields: [
-          TabSpecificField.USERS,
```
Comment (Ashish8689) [inline openmetadata-ui/src/main/resources/ui/src/pages/TeamsPage/TeamsPage.tsx:220]: I believe we can also remove this user field from the `fetchTeamAdvancedDetails` function. Can you verify it once ?

File: openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Teams.spec.ts [hunk 2]
```diff
@@ -562,4 +562,77 @@ test.describe('Teams Page', () => {
...
+    const table1 = new TableClass();
+
+    const team1 = new TeamClass({
+      name: `pw%percent-${id}`,
+      displayName: `pw team percent ${id}`,
+      description: 'playwright team with percent description',
+      teamType: 'Group',
+    });
+
+    await table1.create(apiContext);
```
Comment (Ashish8689) [inline openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Teams.spec.ts:579]: Not sure if we require this, as we are not performing any test around the Entity in this step. Can you verify and remove it if not needed ?

**Review by Ashish8689 — CHANGES_REQUESTED** [review]

File: openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Teams.spec.ts (hunk 2 above)
Comment (pranita09) [reply openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Teams.spec.ts:579]: We require this to create a new team and locate Users tab in that teams page.

File: openmetadata-ui/src/main/resources/ui/src/pages/TeamsPage/TeamsPage.tsx (hunk 1 above)
Comment (pranita09) [reply openmetadata-ui/src/main/resources/ui/src/pages/TeamsPage/TeamsPage.tsx:220]: Verified and removed the field from the function.

**Review by Ashish8689 — APPROVED** [review]

**Review by Ashish8689 — APPROVED** [review]

**Review by sonika-shah — APPROVED** [review]

**Pull request merged**